### DISCUSSION TITLE
discord / community server ?

### DISCUSSION THREAD

**Discussion by thibault-ketterer** [OlivierGaland/CashFactory#6] [Ideas]
why not create a discord server about passive income for linux ?

**Comment by OlivierGaland** [comment]
Hello,

I will unfortunately have no time to do that, maybe in the future if the community grows up, for the moment i will post articles here to give results of my experience on those application (I will have a full month using it in 1 weeks).

Of course this server could be created on anyone initiative, i will x-ref it on this page.

Olivier.

**Reply by MeekOnGithub** [reply]
I can :D
